src/picterra/__main__.py

```diff
@@ -196,6 +199,10 @@ def parse_args(args):
             client = APIClient()
             client.delete_raster(options.raster)
             logger.info('Deleted raster whose id was %s' % options.raster)
+        elif options.delete == 'detector':
+            client = APIClient()
```

Comment:
It looks like we're creating `APIClient` on every branch here, perhaps we could just create it once at the start instead? Otherwise I feel like that's kind of adding noise everywhere.

Reply:
Right! Checked